Doc/tools/extensions/pyspecific.py

```diff
@@ -58,7 +58,7 @@ def new_depart_literal_block(self, node):
 
 def issue_role(typ, rawtext, text, lineno, inliner, options={}, content=[]):
     issue = utils.unescape(text)
-    text = 'issue ' + issue
+    text = 'bpo- ' + issue
```

Comment:
No, there shouldn't be a space :sweat_smile:
Thanks for the review :) Will fix soon.